src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -137,19 +138,27 @@ private static void ReinitializeAppSettings()
             }
         }
 
-        private static ScriptHostConfiguration GetScriptHostConfiguration(string scriptPath, string logPath)
```

Comment:
I believe actually most of these changes here were added by @fabiocav if I'm not mistaken.
https://github.com/Azure/azure-webjobs-sdk-script/commit/6b767e1022c9f31b2be0e2a842fceb0084c934c4